benchmark/csrc/sampler/mapper.cpp

```diff
@@ -0,0 +1,85 @@
+#include <algorithm>
+#include <random>
+#include <type_traits>
+#include <vector>
+
+#include <benchmark/benchmark.h>
+
+#include "pyg_lib/csrc/sampler/cpu/mapper.h"
+
+template <typename node_t, typename scalar_t>
+class BenchmarkMapperCreationAndInsertion : public benchmark::Fixture {
+  static_assert(std::is_integral<node_t>::value &&
```

Comment:
Do we need this check? I think we can also benchmark `node_t` without being necessarily an integral.

Reply:
As in below [comment](https://github.com/pyg-team/pyg-lib/pull/101#discussion_r976057274), this is needed to have a single case which works on both map and vector under the hood.